lib/commands/support.ts

```diff
@@ -130,8 +130,11 @@ export default class SupportCmd extends Command {
 			ux.action.stop();
 		}
 
+		const { getFleetSlug } = await import('../utils/sdk');
+
 		// Process applications
-		for (const appName of appNames) {
+		for (let appName of appNames) {
+			appName = await getFleetSlug(balena, appName);
```

Comment:
NIT: that's a bit misleading of what it holds, right?

Reply:
Do you mean the variable names? That it should be either of the following? -

* `fleetSlug = await getFleetSlug(balena, appName);`
* `appSlug = await getAppSlug(balena, appName);`

Reply:
Yep app/fleetSlug sounds more appropriate :+1:

Reply:
Done in #2420.